target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -57,6 +57,11 @@ platform_do_upgrade() {
 		MTD_CONFIG_ARGS="-s 0x200000"
 		default_do_upgrade "$ARGV"
 		;;
+	vr2600v)
```

Comment:
didn't fixed we this alphabetical distortion already? Anyway, please keep alphabetical order.